### DISCUSSION TITLE
How to tighten the template to just what I need?

### DISCUSSION THREAD

**Discussion by thamas** [arthelokyo/astrowind#447] [Q&A] [answered]
AstroWind provides a lot of pages, components and features out of the box. Thanks for all the work!

I've just installed it and I'm curious about the best way to keep just what I need. 

To be honest, I'd like to have the blog listing as the homepage, the blog-related features (tags, categories) and some pages (like an "About" page) only. 

How would you do that?

**Comment by thamas** [comment]
OK I have some ideas now:

- Most likely I do not need to delete anything – everything can be kept eg. to be used as an examplewhen needed.
- Basic configuration is understandable and easy to modify. 
- I can make the blog listing the homepage by renaming/removing src/pages/index.astro and setting the blog list path to '' (It is discussed here: https://github.com/onwidget/astrowind/discussions/172 – though it is said not to work on GitHub pages…?? What I know currently that it works on local.)
- I still need to remove a lot of menu items in the site header and at the bottom of the pages. I got to check their source(s). Also I have to check how the different header menu variations are created.

**Comment by prototypa** [comment, marked as answer]
Hi @thamas 

We are working on version 2 with a generator to make all this easier. For now, you will have to remove what you don't need.

By the way, check out this comment where it is clarified how to put the blog on the homepage:
https://github.com/onwidget/astrowind/discussions/172#discussioncomment-9439544

**Reply by thamas** [reply]
Thanks, @prototypa, I change it step by step. It was more scary at first sight than it is when one checks the details. 🙂
